src/Components/FolderMain/FolderMain.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useState, useEffect } from "react";
+import Cards from "../Cards/Cards";
+import Article from "../Article/Article";
+import styles from "./FolderMain.module.css";
+import { FolderDataAll, FolderData } from "../../api/parseData";
+import { useFetch } from "../../hooks/useFetch";
+import { BASE_URL } from "../../constants/baseURL";
+import Modal from "../Modal/Modal";
+import Image from "next/image";
+
+interface Card {
+    id: string;
+    url: string;
+    showStar?: boolean;
+    image_source?: string;
+    title?: string;
+    description?: string;
+    created_at: string;
+}
+
+interface Link {
+    id: string;
+    created_at: string;
+    name: string;
+    user_id: number;
+    favorite: boolean;
+    link: {
+        count: number;
+    };
+}
+
+const ALL_FOLDERS: string = "전체";
```

Comment:
타입스크립트는 할당되는 값을 통해 타입을 추론하기 때문에 이 경우에는 string 이라는 타입을 알려주지 않아도 알아서 string 타입으로 인식합니다

```suggestion
const ALL_FOLDERS = "전체";
```

참고: [타입 추론](https://typescript-kr.github.io/pages/type-inference.html)